lib/screens/calc_screen.dart

```diff
@@ -0,0 +1,175 @@
+import 'package:flutter/material.dart';
+import 'package:calculator/widgets/button.dart';
+import 'package:calculator/utils/constants.dart';
+
+class CalcScreen extends StatelessWidget {
+  const CalcScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) => Scaffold(
+        backgroundColor: const Color.fromRGBO(241, 242, 243, 1),
+        body: Column(
+          children: [
+            Center(
+              child: Padding(
+                padding: const EdgeInsets.only(top: 35),
+                child: SizedBox(
+                  width: Constants.buttonWidth,
+                  height: Constants.buttonHeight,
+                  child: Image.asset(Constants.lightModeAssetPath),
+                ),
+              ),
+            ),
+            const Padding(
+              padding: EdgeInsets.fromLTRB(0, 40, 20, 0),
+              child: Align(
+                alignment: Alignment.bottomRight,
+                child: Text(
+                  '6,291/5',
+                  style: TextStyle(fontSize: 45, color: Colors.grey),
+                ),
+              ),
+            ),
+            const Padding(
+              padding: EdgeInsets.fromLTRB(0, 16, 20, 0),
+              child: Align(
+                alignment: Alignment.bottomRight,
+                child: Text(
+                  '1,258.2',
+                  style: TextStyle(fontSize: 96, color: Colors.black),
+                ),
+              ),
+            ),
+            Padding(
+              padding: const EdgeInsets.fromLTRB(24, 16, 20, 0),
+              child: Align(
+                alignment: Alignment.bottomCenter,
+                child: Column(
+                  children: [
+                    Row(
```

Comment:
tutaj dwie rzeczy, pierwsza ten Row wydzielasz do metody
i druga wywalasz te SizedBoxy i dajesz `        mainAxisAlignment: MainAxisAlignment.spaceBetween,
` czyli powinno to wyglądać tak
`
  Row rowButtons(List<Widget> buttons) => Row(
        mainAxisAlignment: MainAxisAlignment.spaceBetween,
        children: buttons,
      );
`

Reply:
A czy jest sposób żeby zrobić tak samo ale między wierszami?

Reply:
w sumie to musiałbyś na sztywno ustawić chyba wysokość tej kolumny, ewentualnie pobawić się z Expanded i wtedy możesz to zrobić tak samo i wyrzucić wtedy te 16